src/Features/CSharp/Portable/UseLocalFunction/CSharpUseLocalFunctionCodeFixProvider.cs

```diff
@@ -120,8 +120,8 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
 
             var invokeMethod = delegateType.DelegateInvokeMethod;
 
-            var returnType = invokeMethod.ReturnsVoid
-                ? s_voidType
+            var returnType = invokeMethod.ReturnsByRef ? invokeMethod.ReturnType.GenerateRefTypeSyntax()
+                : invokeMethod.ReturnsByRefReadonly ? invokeMethod.ReturnType.GenerateRefReadOnlyTypeSyntax()
                 : invokeMethod.ReturnType.GenerateTypeSyntax();
```

Comment:
better to look like

```c#
var returnType = invokeMethod.ReturnsByRef ? invokeMethod.ReturnType.GenerateRefTypeSyntax() : invokeMethod.ReturnsByRefReadonly 
                                           ? invokeMethod.ReturnType.GenerateRefReadOnlyTypeSyntax() : invokeMethod.ReturnType.GenerateTypeSyntax();

Reply:
IMHO, putting the condition always on the  left handside makes it more readable. In the following:
```csharp
var returnType = invokeMethod.ReturnsByRef ? invokeMethod.ReturnType.GenerateRefTypeSyntax() : invokeMethod.ReturnsByRefReadonly 
                                           ? invokeMethod.ReturnType.GenerateRefReadOnlyTypeSyntax() : invokeMethod.ReturnType.GenerateTypeSyntax();
```
The first line will read to me as a complete sentence, in which `invokeMethod.ReturnsByRefReadonly  returns the false result, and it will be separate from the following line.

---
In reply to: [141180523](https://github.com/dotnet/roslyn/pull/22285#discussion_r141180523) [](ancestors = 141180523)

Reply:
refactored

---
In reply to: [141180523](https://github.com/dotnet/roslyn/pull/22285#discussion_r141180523) [](ancestors = 141180523)